cmux-tui/scripts/smoke-windows-conpty-resize.py

```
@@ -69,29 +82,157 @@ def wait_for_socket(
-    if result is not None:
-        raise result
+def wait_for_tui_start(reader: PtyOutputReader) -> str:
+    output = ""
+    deadline = time.monotonic() + TIMEOUT_SECONDS
+    post_probe_marker = "\x1b[>1s"
+    startup_complete = False
+    try:
+        while post_probe_marker not in output:
+            remaining = deadline - time.monotonic()
+            if remaining <= 0:
+                raise TimeoutError(f"attach did not finish terminal setup: {output!r}")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)